patches/minecraft/net/minecraft/entity/passive/EntityHorse.java.patch

```diff
@@ -1,6 +1,40 @@
 --- ../src-base/minecraft/net/minecraft/entity/passive/EntityHorse.java
 +++ ../src-work/minecraft/net/minecraft/entity/passive/EntityHorse.java
-@@ -405,7 +405,7 @@
+@@ -77,6 +77,7 @@
+     private static final DataParameter<Integer> field_184789_bG = EntityDataManager.<Integer>func_187226_a(EntityHorse.class, DataSerializers.field_187192_b);
+     private static final DataParameter<Optional<UUID>> field_184790_bH = EntityDataManager.<Optional<UUID>>func_187226_a(EntityHorse.class, DataSerializers.field_187203_m);
+     private static final DataParameter<Integer> field_184791_bI = EntityDataManager.<Integer>func_187226_a(EntityHorse.class, DataSerializers.field_187192_b);
++    private static final DataParameter<String> ARMOR_STRING = EntityDataManager.<String>func_187226_a(EntityHorse.class, DataSerializers.field_187194_d);
```

Comment:
as long as new DataParameter's are last (aka we do not interfere with the ordering of the original ones), they will be fine.

Forge adds new DataParameter's to Villagers (or was it zombies?) to support custom variants